app/helpers/substring-projectbrief.js

```diff
@@ -0,0 +1,18 @@
+import { helper } from '@ember/component/helper';
+
+
+function shortenProjectBrief([projectbrief]) {
```

Comment:
We'll need to figure out how to get JS linting working on your text editor

Reply:
I cleaned up the extra whitespace. Don't see any linting errors tho. Gonna merge this in… 